src/web/src/list_gpio.ecpp

```diff
@@ -138,9 +154,6 @@ get_list (std::string &output)
         zmsg_destroy(&resp);
         return -5;
     }
-
-    zmsg_destroy(&resp);
```

Comment:
What happens `if (!result)` ? Shouldn't the removed destroy and return remain in place?